### ISSUE TITLE
"Not in Use" filter

### ISSUE THREAD

**Issue by schellenbergk** [sanity-io/sanity-plugin-media#150]
**Describe the solution you'd like**
It would be nice to include "Not in Use" filter for media to be able to cleanup media from unused images.

**Comment by JamesSingleton** [comment]
Yes this would be great! Trying to cleanup images that aren't being referenced to decrease my assets footprint.

**Comment by robinpyon** [comment]
Hey @schellenbergk – I believe this is already possible by using the existing `In Use` filter and setting the value to `False`. This should show all assets that aren't being referenced elsewhere in your dataset.

Does this accomplish what you're after, or perhaps I'm misunderstanding the ask here?

**Comment by JamesSingleton** [comment]
@robinpyon this accomplishes what I wanted at least 😅

**Comment by JamesSingleton** [comment]
Wow that filter is more reliable than I am... Apparently I have uploaded duplicate photos for some reason/ some how...

**Comment by AlexanderBMAS** [comment]
In Use = false starts to fail / timeout when there are large asset libraries eg 25GB+

**Comment by AbeCole** [comment]
I'm looking for an "exclusion" option. For example tagging an asset as "Hidden" (generally this would be done when uploading an asset automatically using the Sanity API) and then have these assets hidden from the Media browser by default (after setting a config option).
